src/service/FileTransferService.ts

```diff
@@ -1,14 +1,18 @@
 import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
 import FormData from 'form-data';
-import { CREATED, UNSUPPORTED_MEDIA_TYPE } from 'http-status-codes';
+import { CREATED, NOT_FOUND, UNSUPPORTED_MEDIA_TYPE } from 'http-status-codes';
 
 import { loggerInstance } from 'app/middleware/Logger';
 
 export class FileTransferService {
 
     constructor(private readonly url: string, private readonly key: string) {
-        this.url = url;
-        this.key = key;
+        if (url == null) {
```

Comment:
I'v noticed that you have been using '==' rather than '===' throughout your implementation, what is the reason behind it?  From my understanding the triple equals compared both the type and the value, and is therefore more strict. Shouldn't we by default use it?